dask_bigquery/core.py

```diff
@@ -174,3 +200,108 @@ def make_create_read_session_request(row_filter="", max_stream_count=0):
 
         graph = HighLevelGraph({output_name: layer}, {output_name: set()})
         return new_dd_object(graph, output_name, meta, divisions)
+
+
+def to_gbq(
+    df,
+    *,
+    project_id: str,
+    dataset_id: str,
+    table_id: str,
+    gs_bucket: str = "dask-bigquery-tmp",
+    credentials: Credentials = None,
+    delete_bucket: bool = False,
+    parquet_kwargs: dict = None,
+    load_api_kwargs: dict = None,
+):
+    """Write dask dataframe as table using BigQuery Load API.
+    Writes Parquet to GCS for intermediary storage.
+
+    Parameters
+    ----------
+    project_id: str
+      Name of the BigQuery project id.
+    dataset_id: str
+      BigQuery dataset within project
+    table_id: str
+      BigQuery table within dataset
+    gs_bucket: str
+      Google Cloud Storage bucket name, for intermediary parquet storage. If the bucket doesn't
+      exist, it will be created. The account you're using will need Google Cloud Storage
+      permissions (storage.objects.create, storage.buckets.create). If a persistent bucket is used,
+      it might be a good idea to configure it with retention, to make sure the temporary data gets
+      cleaned periodically, even if the job is failed to finish and cleanup.
```

Comment:
nit: I think the documentation should use more explicit language. `might` sounds like we're not sure ourselves but this is clearly a recommendation

```suggestion
      it is recommended to configure a retention policy that ensures
      the data is cleaned up in case of job failures.
```